mac/macdeploy.sh

```diff
@@ -7,6 +7,7 @@
 
 MACDEPLOYQT="macdeployqt"
 LIBRARY_NAME="libQupZilla.2.dylib"
+PLUGINS="QupZilla.app/Contents/Resources/plugins"
```

Comment:
Doesn't work any place else with `install_name_tool`... even if I add an id to it it never finds it and conversely the executable of `QupZilla` nor the lib that QupZilla creates... tries `rpath` too. This is the only location it works at.